src/classes/HiFiAudioAPIData.ts

```diff
@@ -351,6 +351,12 @@ export function eulerFromQuaternion(quat: OrientationQuat3D, order: OrientationE
     return new OrientationEuler3D({ pitchDegrees: RAD_TO_DEG * pitch, yawDegrees: RAD_TO_DEG * yaw, rollDegrees: RAD_TO_DEG * roll });
 }
 
+/**
+ * Instantiations of this class define a map between hashed visit IDs and the gains of other users as percieved by the current user's session. This class is used by {@link HiFiAudioAPIData.otherUserGainQueue} to keep track of which gain changes need to be sent to the server.
+ * See also {@link HiFiCommunicator.setOtherUserGainForThisConnection}, which allows you to set the desired gain for a user and also send the data to the server. If you need to perform moderation actions which apply to all users, use the {@link https://docs.highfidelity.com/rest/latest/index.html|Administrative REST API}.
+ */
+export type OtherUserGainMap = { [key: string]: number };
+
```

Comment:
This map is only used internally and isn't meant to be adjusted or inspected directly by the user (i.e. the developer), correct? Let's call that out explicitly in these comments. Is it possible to make it private and/or otherwise hidden from documentation and developers?

Reply:
I can convert the comment to something that won't show up in the documentation, and emphasize that it is internal.

Reply:
That sounds excellent, thank you!